tensorboard/plugins/histogram/histograms_plugin_test.py

```diff
@@ -173,6 +175,33 @@ def test_histograms_with_histogram(self):
     self._test_histograms(self._RUN_WITH_HISTOGRAM,
                           '%s/histogram_summary' % self._HISTOGRAM_TAG)
 
+  def _test_histograms_csv(self, run_name, tag_name, should_work=True):
+    self.set_up_with_runs([self._RUN_WITH_LEGACY_HISTOGRAM,
+                           self._RUN_WITH_HISTOGRAM])
+    if should_work:
+      (data, mime_type) = self.plugin.histograms_impl(
+        tag_name, run_name,
+        output_format=histograms_plugin.OutputFormat.CSV)
+      self.assertEqual('text/csv', mime_type)
+      s = StringIO(data)
+      reader = csv.reader(s)
+      self.assertEqual(['Wall time', 'Step', 'BinStart',
+                        'BinEnd', 'BinValue'], next(reader))
+      self.assertEqual(len(list(reader)), self._STEPS)
+    else:
+      with self.assertRaises(KeyError):
+        self.plugin.histograms_impl(
+          self._HISTOGRAM_TAG, run_name,
+          output_format=histograms_plugin.OutputFormat.CSV)
+
+  def test_histograms_csv_with_legacy_histograms(self):
+    self._test_histograms_csv(self._RUN_WITH_LEGACY_HISTOGRAM,
+                              self._LEGACY_HISTOGRAM_TAG)
+
+  def test_histograms_csv_with_histogram(self):
+    self._test_histograms_csv(self._RUN_WITH_HISTOGRAM, self._HISTOGRAM_TAG,
```

Comment:
> ```python
>     self._test_histograms_csv(self._RUN_WITH_HISTOGRAM, self._HISTOGRAM_TAG,
>                               should_work=False)
> ```

This indeed fails, but not for the right reason. A histogram run with a
histogram tag _should_ work, but you’ve specified the wrong tag: it
should be `"%s/histogram_summary" % self._HISTOGRAM_TAG`, as listed in
the `test_histograms_with_histogram` test case above.

Please add a test case that actually should fail (e.g., requesting a
scalar tag on a histograms run).